src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -0,0 +1,295 @@
+---
+title: "How to use NRQL: the mechanics of querying"
+tags:
+  - Query your data
+  - 'NRQL: New Relic query language'
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query with NRQL, NRQL syntax, and how you can explore your data."
+redirects:
+---
+
+import queriesnrqlEventDefinitionsQueryBuilder from 'images/queries-nrql_screenshot-crop_event-definitions-query-builder.png'
+
+import queriesnrqlAttributeDefinitionsQueryBuilder from 'images/queries-nrql_screenshot-crop_attribute-definitions-query-builder.png'
+
+import queriesnrqlViewQueryforChart from 'images/queries-nrql_screenshot-crop_view-query-for-chart.png'
+
+import queriesnrqlNrqlExampleTimeseries from 'images/queries-nrql_screenshot-crop_nrql-example-timeseries.png'
+
+import queriesNrqlBasicExample from 'images/queries-nrql_screenshot-crop_basic-nrql-query-example.png'
+
+Before using any tool, it's best to know how to use it, and NRQL is no exception to that rule. There's a process to creating queries with NRQL, and how to structure and write NRQL queries. Understanding the rules of querying with NRQL enables you to get the most from your data. Even if you've never queried anything before, once you understand the basics you can find ways to access (almost) any data you need and visualize them in [charts](/docs/query-your-data/explore-query-data/use-charts/use-your-charts) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
```

Comment:
```suggestion
Before using any tool, it's best to know how to use it. There's a process to creating queries with NRQL, and how to structure and write NRQL queries. Understanding the rules of querying with NRQL enables you to get the most from your data. Even if you've never queried anything before, once you understand the basics you can find ways to access (almost) any data you need and visualize them in [charts](/docs/query-your-data/explore-query-data/use-charts/use-your-charts) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
```

Reply:
Thank you! I cut down links in some places, I'm hoping that in these docs we can give the reader the information they need to understand NRQL but give them access to more in-depth information when they're ready. I will be using our monitoring tools to keep tabs on this for sure and adjust as we go.

Reply:
I also added the change you suggested.